### PR TITLE
Call updateReportStatus and notification API as Kibana user

### PR DESCRIPTION
**Pull request by zhongnansu**
*Issue #, if available:*

*Description of changes:*
- use `callAsInternalUser` for `updateReportState` and `notification.send`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: kibana-reports/server/routes/lib/deliverReport.ts [hunk 1]
```diff
@@ -61,27 +61,26 @@ export const deliverReport = async (
...
-      htmlDescription: template,
+      htmlDescription: embeddedHtml,
       refTag: reportId,
     };
 
     // send email
-    const notificationResp = await callCluster(
-      notificationClient,
+    const notificationResp = await notificationClient.callAsInternalUser(
+      // @ts-ignore
```
Comment (davidcui-amzn) [inline kibana-reports/server/routes/lib/deliverReport.ts:79]: What's the warning that this `@ts-ignore` is silencing?

File: kibana-reports/server/routes/utils/visualReportHelper.ts [hunk 2]
```diff
@@ -87,11 +88,11 @@ export const createVisualReport = async (
   let element: any;
   // crop content
   if (reportSource === REPORT_TYPE.dashboard) {
-    await page.waitForSelector('#dashboardViewport');
-    element = await page.$('#dashboardViewport');
+    await page.waitForSelector(SELECTOR.dashboard);
```
Comment (davidcui-amzn) [inline kibana-reports/server/routes/utils/visualReportHelper.ts:91]: Nice change

**Review by davidcui-amzn — APPROVED** [review]
LGTM- just one minor question

**Review by joshuali925 — APPROVED** [review]

File: kibana-reports/server/routes/lib/deliverReport.ts (hunk 1 above)
Comment (zhongnansu) [reply kibana-reports/server/routes/lib/deliverReport.ts:79]: overload error, but as a matter of fact, both callAsInternal user and callAsCurrentUser can take a string

**Pull request merged**